src/views/bootablevolumes/utils/utils.ts

```diff
@@ -81,3 +81,24 @@ export const changeBootableVolumeMetadata =
         ],
       }));
   };
+
+interface InstanceTypesAndSizesObject {
```

Comment:
I suggest renaming the type to `InstanceTypesToSizesMap`, too. I think the type should be moved to bootablevolumes/utils/types.